### PR TITLE
Feature/edit

### PR DESCRIPTION
**Pull request by andy-schwedler**
Hi there,

it is now possible to Edit a Memory headline & date on the overview page.

Refactoring is part of the next user story, but for now I would like to keep it this way.

Check it out yourself and let me hear or read about it 😃

### CODE REVIEW

File: components/MemoryCard/MemoryOverviewCard.js [hunk 1]
```diff
@@ -1,28 +1,112 @@
...
+  }
+
+  function handleEditSubmit(event) {
+    event.preventDefault();
+
+    const editedName = event.target.elements.memory.value;
+    const editedDate = event.target.elements.date.value;
+    const editModeInput = { name: editedName, date: editedDate };
+
+    console.log(editedDate);
```
Comment (onebarloop) [inline components/MemoryCard/MemoryOverviewCard.js]: you left something there 😅

File: components/MemoryCard/MemoryOverviewCard.js [hunk 2]
```diff
@@ -1,28 +1,112 @@
...
+              />
+            ) : (
+              <BookmarkButton
+                isFavorite={sampleEvent.isFavorite}
+                id={sampleEvent.id}
+                onToggleFavorite={onToggleFavorite}
+              />
+            )}
+          </StyledMenu>
+        ) : null}
```
Comment (onebarloop) [inline components/MemoryCard/MemoryOverviewCard.js]: You don't need the ternary operator here - it only returns null if the condition is not met. Instead you could rewrite line 74 like this:

`{displayOptionMenu && (...)}`

So if `displayOptionsMenu` equals true, the code before the `:` is executed. You don't need to return null if not.

**Review by onebarloop — APPROVED** [review]
Looks very solid to me. I remember the session when we discussed exactly this! 

Two small annotations but nothing critical. Keep up the good work :)

File: components/MemoryCard/MemoryOverviewCard.js (hunk 1 above)
Comment (andy-schwedler) [reply components/MemoryCard/MemoryOverviewCard.js]: Removed ✅

File: components/MemoryCard/MemoryOverviewCard.js (hunk 2 above)
Comment (andy-schwedler) [reply components/MemoryCard/MemoryOverviewCard.js]: Makes great sense, Alex 🙏

**Pull request merged**